packages/boot/test/acceptance/controller.booter.acceptance.ts

```diff
@@ -33,6 +33,7 @@ describe('controller booter acceptance tests', () => {
   });
 
   async function getApp() {
+    await sandbox.copyFile(resolve(__dirname, '../fixtures/package.json'));
```

Comment:
I think this will not work after the changes  I made in https://github.com/strongloop/loopback-next/pull/1824) - the `package.json` file is not going to be copied over from `test` to `dist/test`.

I think the easiest solution is to load the package json file from `test/fixtures/application.ts`:

```ts
import * as pkg from './package.json'
```